botEngine.js

```diff
@@ -55,7 +55,7 @@ async function listenToMessages(client) {
       if (creationsMessage) {
         creationsMessage.delete();
       }
-      creationsMessage = await message.channel.send('Reminder: This channel is for posting links to your creations only. You can discuss the projects posted here in the sibling channel <#634025871614803968>');
+      creationsMessage = await message.channel.send('Reminder: This channel is for posting links to your creations only. You can discuss the projects posted here in the sibling channel <#634025871614803968> React to a project you\'ve reviewed with :white_check_mark:. Try to review projects that haven\'t been reviewed yet!');
```

Comment:
Nitpick, there should be a period between these sentences.

Reply:
Is it ok now?